test-suite/exonum-py-tests/exonum_tests/deploy.py

```diff
@@ -307,32 +237,121 @@ def test_deploy_regular_stop_running_instance(self):
             client = ExonumClient(host, public_port, private_port)
             available_services = client.public_api.available_services().json()
             # crypto instance always first element in array
-            self.assertEqual(available_services['services'][0]['status']['type'], 'stopped')
+            self.assertEqual(
+                available_services["services"][0]["status"]["type"], "stopped"
+            )
             with ExonumCryptoAdvancedClient(client) as crypto_client:
                 alice_keys = KeyPair.generate()
                 tx_response = crypto_client.create_wallet(
                     alice_keys, "Alice" + str(validator_id)
                 )
                 # in case of stopped service its tx will not be processed
                 self.assertEqual(tx_response.status_code, 400)
-                self.assertIn("Specified service is not active", str(tx_response.content))
+                self.assertIn(
+                    "Specified service is not active", str(tx_response.content)
+                )
+
+        # resume service
+        instances = {"crypto": {"artifact": "cryptocurrency", "action": "resume"}}
+        cryptocurrency_advanced_config_dict = generate_config(
+            self.network, instances=instances
+        )
+
+        cryptocurrency_advanced_config = Configuration(
+            cryptocurrency_advanced_config_dict
+        )
+        with Launcher(cryptocurrency_advanced_config) as launcher:
+
+            launcher.deploy_all()
+            launcher.wait_for_deploy()
+            launcher.start_all()
+            launcher.wait_for_start()
+
+        for validator_id in range(self.network.validators_count()):
+            host, public_port, private_port = self.network.api_address(validator_id)
+            client = ExonumClient(host, public_port, private_port)
+            available_services = client.public_api.available_services().json()
+            # crypto instance always first element in array
```

Comment:
IMHO it's a bit fragile. I'd suggest to create a helper function (e.g. `find_active_service(available_service, service_name)`), which finds required service by the name and returns its status.